src/main/java/com/minecolonies/coremod/colony/buildings/modules/TavernBuildingModule.java

```diff
@@ -173,7 +175,11 @@ private void spawnVisitor()
 
         newCitizen.getCitizenSkillHandler().init(recruitLevel);
 
-        BlockPos spawnPos = BlockPosUtil.findAround(building.getColony().getWorld(), building.getPosition(), 1, 1, bs -> bs.getMaterial() == Material.AIR);
```

Comment:
I wonder if we want a util for this "findCitizenSpawnPos" and use that then everywhere, also for recall etc

Reply:
ye we could add some predefined to the utils for common cases